app/models/storage.rb

```diff
@@ -581,12 +581,12 @@ def used_space
   alias_method :v_used_space, :used_space
 
   def used_space_percent_of_total
-    total_space.to_f == 0.0 ? 0.0 : (used_space.to_f / total_space.to_f * 1000.0).round / 10.0
+    total_space.to_f.zero? ? 0.0 : (used_space.to_f / total_space * 1000.0).round / 10.0
```

Comment:
Do we need to cast `total_space.to_f` if we're just going to check for `.zero?` ?  Wonder if `total_space.zero? ? 0.0 : (used_space.to_f / total_space * 1000.0).round / 10.0` would be cleaner

Reply:
Agreed, updated.